plugins/statistics.koplugin/main.lua

```diff
@@ -344,6 +344,29 @@ end
 --          weekly - show weekly
 --          monthly - show monthly
 function ReaderStatistics:getDatesFromAll(sdays, ptype)
+    local shortDayOfWeekTranslation = {
```

Comment:
Better declare these tables as a module level variable so they are not recreated everytime this function is called.